### PR TITLE
Sptribs cic release 2 1 merge 2

### PR DESCRIPTION
**Pull request by osheaf01**
### Change description ###

Merging changes from sptribs_cic_release_2 to sptribs_cic_release_2_1

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java [hunk 1]
```diff
@@ -115,23 +115,24 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(
         DynamicList dynamicList = caseData.getCicCase().getDraftOrderDynamicList();
         UUID code = dynamicList.getValue().getCode();
         String label = dynamicList.getValue().getLabel();
-        String[] dynamicListLabel = label.split(DOUBLE_HYPHEN);
```
Comment (ElifDenizEsea) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java:118]: this part will be removed no need to merge

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/PartiesCICWithOutSubject.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.sptribs.ciccase.model;
+
```
Comment (ElifDenizEsea) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/PartiesCICWithOutSubject.java:2]: I do not think this file is used

**Review by ElifDenizEsea — CHANGES_REQUESTED** [review]

**Review by muradalamv1 — APPROVED** [review]

**Review by geciciserkan — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/PartiesCICWithOutSubject.java (hunk 2 above)
Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/PartiesCICWithOutSubject.java]: Removed

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java (hunk 1 above)
Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java]: Reverted

**Review by ElifDenizEsea — APPROVED** [review]

**Pull request merged**